apisix/plugins/limit-count/limit-count-redis-cluster.lua

```diff
@@ -112,5 +113,5 @@ function _M.incoming(self, key)
     return 0, remaining, ttl
 end
 
-
+_M.script = script
```

Comment:
Why export this?

Reply:
The redis script in `limit-count-redis.lua` and `limit-count-redis-cluster.lua` are the same so I exported it to avoid duplication.